app/main.py

```diff
@@ -1,7 +1,25 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import requests
+import os
+
+
+def get_weather(api_key: str) -> None:
+    city = "Paris"
+    base_url = f"http://api.weatherapi.com/v1/current.json?key=" f"{api_key}&q={city}"
```

Comment:
Code Style Issue: It's a good practice to define constants for values that are not going to change, such as the base URL for the API. This makes the code easier to read and maintain. Please consider using a constant for the base URL.